src/haddock/gear/prepare_run.py

```diff
@@ -106,52 +115,100 @@ def setup_run(workflow_path, restart_from=None):
         Whether to erase the previous run folder and reprare from
         scratch. Defaults to `True`.
 
+    start_from_copy : str or Path
+        The path created with `haddock3-copy` to start the run from.
+        Defaults to None, do not use this option.
+
     Returns
     -------
     tuple of two dicts
         A dictionary with the parameters for the haddock3 modules.
         A dictionary with the general run parameters.
     """
+    from_scratch = restart_from is None and start_from_copy is None
```

Comment:
Maybe it is better to move this to just before where it is used (just before line 153 according to Github).